proxy/README.md

```diff
@@ -0,0 +1,12 @@
+# Keycloak Security Proxy
+
+The Keycloak Security Proxy docker image.
+
+## Usage
+
+This image is intended to be used with an existing `/tmp/conf/proxy.json` on the host machine. 
+See [the proxy documentation](https://keycloak.gitbooks.io/documentation/server_installation/topics/proxy.html) 
+for more details.
```

Comment:
@jpkrohling removed and *same error*. IMO, we should clearly state what is necessary for people to run it.

If it just run `docker run --rm --name keycloak-proxy -v /tmp/conf:/opt/jboss/conf -p 8180:8180 jpkroehling/keycloak-proxy`. The README at least should be clear about the keystore generation or how `proxy.json` should look like. Otherwise, I'm 100% sure that people will come up with questions.

The blog post is great and I'm happy to read, but the README must contain the minimum for people to get started without having to go to your blog post.